web3_proxy/src/app/mod.rs

```diff
@@ -110,7 +152,7 @@ pub struct AuthorizationChecks {
     /// IMPORTANT! Once confirmed by a miner, they will be public on the blockchain!
     pub private_txs: bool,
     pub proxy_mode: ProxyMode,
-    pub balance: Option<Decimal>,
+    pub user_tier: UserTier,
```

Comment:
Why is this needed? We don't generally care about having the tier. We just want the limits, and those are already part of this cached data

Reply:
removed it, was an artefact of me playing around with the cache